### PR TITLE
remove data ingest, update batch and serving examples to be LLMs

### PR DESCRIPTION
**Pull request by angelinalg**
## Why are these changes needed?
Removing Data Ingest and replacing it with Batch Inference. (Pendo click data seems to indicate that Batch Inference is resonating more than Data Ingest, too.)
Updating Batch Inference code snippet to use Data vs BatchPredictor. 
Updating Serve code snippet to be an LLM example.
Updated corresponding notebook links.

## Checks

- [ x] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by amogkam — APPROVED** [review]
lgtm

File: doc/source/index.md [hunk 1]
```diff
@@ -182,50 +200,46 @@ result_grid = tuner.fit()
...
+            …
+        )
+        return pd.DataFrame(
+            self.tokenizer.batch_decode(gen_tokens), columns=["responses"]
+        )
+
+    async def __call__(self, http_request: Request) -> str:
+        json_request: str = await http_request.json()
+        prompts = []
+        prompts: list[str] = await http_request.json()["prompts"]
```
Comment (amogkam) [inline doc/source/index.md:239]:
```suggestion
        prompts: list[str] = await http_request.json()["prompts"]
```

File: doc/source/index.md [hunk 2]
```diff
@@ -182,50 +200,46 @@ result_grid = tuner.fit()
...
+           return self.generate(prompts)
```
Comment (amogkam) [inline doc/source/index.md:240]: I think this should be unindented

File: doc/source/index.md [hunk 3]
```diff
@@ -182,50 +200,44 @@ result_grid = tuner.fit()
...
+            input_ids,
+            …
+        )
+        return pd.DataFrame(
+            self.tokenizer.batch_decode(gen_tokens), columns=["responses"]
+        )
+
+    async def __call__(self, http_request: Request) -> str:
+        prompts: list[str] = await http_request.json()["prompts"]
+           return self.generate(prompts)
```
Comment (amogkam) [inline doc/source/index.md:238]:
```suggestion
        return self.generate(prompts)
```

File: doc/source/index.md (hunk 2 above)
Comment (angelinalg) [reply doc/source/index.md]: ack. accepted your suggestion

**Review by sriram-anyscale — APPROVED** [review]

**Review by matthewdeng — APPROVED** [review]

**Pull request merged**